src/index.js

```diff
@@ -117,21 +103,14 @@ function main(client, options) {
 	});
 }
 
-function EvolvAssetManager(options) {
-	const client = new Evolv(options);
+function EvolvAssetManager(client) {
```

Comment:
If we're attaching the client to the window whats the reasoning behind passing it in here instad of just accessing it globally?

Reply:
This won't only be consumed by the webloader. It will also be instantiated directly. It can't assume it has access to it globally.